### PR TITLE
Aleksei/transactions chunk loading

### PR DESCRIPTION
**Pull request by iambeone**
Closes #ISSUE

**Description:**

On Activity page firstly load some transactions and after scroll end reached load some more

Thank you! 🚀

---

For contributor:

- [ ] Added changes entries. Run `yarn changelog` for a guided process.
- [ ] Reviewed `Files changed` in the github PR explorer
- [ ] Attach screenshots of the UI components on the PR description (if applicable)
- [ ] Scope of work approved for big PRs

For reviewer:

- [ ] Manually tested the changes on the UI

### CODE REVIEW

File: src/components/wallet/PageTransactions.vue [hunk 1]
```diff
@@ -102,51 +117,22 @@ export default {
...
         return {
           networkId: this.network,
-          address: this.address
+          address: this.address,
+          block: this.block
         }
       },
-      update: result => {
+      update(result) {
+        let trx = []
```
Comment (faboweb) [inline src/components/wallet/PageTransactions.vue]: no abbreviations

**Comment by faboweb** [comment]
unit tests are failing

**Comment by iambeone** [comment]
> unit tests are failing

fixed

**Comment by faboweb** [comment]
codecov is at 30%

**Pull request merged**